src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/integration/DraftAdjudicationIntTest.kt

```diff
@@ -493,6 +495,44 @@ class DraftAdjudicationIntTest : IntegrationTestBase() {
     assertThat(draft.get().offenceDetails).hasSize(0)
   }
 
+  @Test
+  fun `add damages to the draft adjudication`() {
+    val testAdjudication = IntegrationTestData.ADJUDICATION_1
+    val intTestData = integrationTestData()
+    val intTestBuilder = IntegrationTestScenarioBuilder(intTestData, this)
+
+    val intTestScenario = intTestBuilder
+      .startDraft(testAdjudication)
+      .setApplicableRules()
+      .setIncidentRole()
+      .setOffenceData()
+
+    val draftId = intTestScenario.getDraftId()
+
+    webTestClient.put()
+      .uri("/draft-adjudications/$draftId/damages")
+      .headers(setHeaders())
+      .bodyValue(
+        mapOf(
+          "damages" to listOf(
+            DamageRequestItem(
+              code = DamageCode.CLEANING, details = "details"
+            )
+          ),
+        )
+      )
+      .exchange()
+      .expectStatus().isCreated
+      .expectBody()
+      .jsonPath("$.draftAdjudication.id").isNumber
+      .jsonPath("$.draftAdjudication.damages[0].code")
+      .isEqualTo(DamageCode.CLEANING.name)
+      .jsonPath("$.draftAdjudication.damages[0].details")
+      .isEqualTo("details")
+      .jsonPath("$.draftAdjudication.damages[0].reporter")
+      .isEqualTo("ITAG_USER")
+  }
```

Comment:
would also recommend also doing a "GET" to prove damages has actually been saved